php-fpm-healthcheck

```diff
@@ -61,7 +61,7 @@ get_fpm_status() {
     if test "$VERBOSE" = 1; then printf "Trying to connect to php-fpm via: %s\\n" "$1"; fi;
     
     # Since I cannot use pipefail I'll just split these in two commands
-    FPM_STATUS=$(env -i REQUEST_METHOD="$REQUEST_METHOD" SCRIPT_NAME="$SCRIPT_NAME" SCRIPT_FILENAME="$SCRIPT_FILENAME" cgi-fcgi -bind -connect "$1" 2> /dev/null)
+    FPM_STATUS=$(env -i REQUEST_METHOD="$REQUEST_METHOD" SCRIPT_NAME="$SCRIPT_NAME" SCRIPT_FILENAME="$SCRIPT_FILENAME" $(which cgi-fcgi) -bind -connect "$1" 2> /dev/null)
```

Comment:
At the beginning of the script there is a `command -v cgi-fcgi` which yields the full path, maybe you can put it inside a variable in case it does exist, this way it remains posix compliant wince `which` is unreliable depending on the system!

Reply:
Good point about which vs command.  Command is much better indeed!

I'll make some changes and you can see if that looks OK

Reply:
I have submitted a proposed change, I believe iot should be ok...

Reply:
It is, thanks so much!